pkg/kbld/resources/file_sources.go

```diff
@@ -24,7 +24,7 @@ func NewStdinSource() StdinSource { return StdinSource{} }
 func (s StdinSource) Description() string { return "stdin" }
 
 func (s StdinSource) Bytes() ([]byte, error) {
-	return ioutil.ReadAll(os.Stdin)
+	return io.ReadAll(os.Stdin)
```

Comment:
Here also should we use the `os` package only as it is already imported so we can ignore importing `io`?

Reply:
oh! my bad. We are good then.